packages/next-sdk/page-tools/page-agent-mask/SimulatorMask.module.css

```
@@ -0,0 +1,14 @@
+.webmcp-page-agent-wrapper {
+	position: fixed;
+	inset: 0;
+	z-index: 2147483641;
+	/* 确保在所有元素之上，除了 panel */
+	cursor: wait;
+	overflow: hidden;
+
+	display: none;
```

Stylelint (17.14.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)